assets/scss/components/stencil/optimizedCheckout/_optimizedCheckout.scss

```diff
@@ -0,0 +1,104 @@
+// =============================================================================
+// Stencil Checkout  - Customize the Optimized Single-Page Checkout experience
+// =============================================================================
+
+// Header
+.checkoutHeader {
+    background-image: url(stencilImage('optimizedCheckout_custom_background_image', 'optimizedCheckout_custom_background_image_size'));
+    background-size: cover;
+    background-color: stencilColor("optimizedCheckout_header_backgroundColor");
+
+    // Desktop Only
+    @include breakpoint("medium") {
+        text-align: stencilString("optimizedCheckout_logo_position");
+    }
+}
+
+// Order Summary, Mobile Order Summary, Modal Order Summary
+.cart,
+.cartDrawer,
+.cart-modal-body,
+.cart-modal-header {
+    background-color: stencilColor("optimizedCheckout_cart_backgroundColor");
+}
+
+// Headings
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+    color: stencilColor("optimizedCheckout_color-headings");
+    font-family: stencilFontFamily("optimizedCheckout_headings_font"), Arial, Helvetica, sans-serif;
```

Comment:
Do we have a naming convention for the config variables? Some variables are delimited by `-`, some are `_`. i.e.: `optimizedCheckout_color-headings` vs `optimizedCheckout_headings_font`. We will need to document these variables soon, so it'd be nice to have some consistency.